src/app/features/tech-record/components/tech-record-amend-vrm/tech-record-amend-vrm.component.ts

```diff
@@ -98,13 +86,11 @@ export class AmendVrmComponent implements OnDestroy {
 
   get makeAndModel(): string {
     const c = this.techRecord;
-    if (!c?.make && !c?.chassisMake) return '';
-
-    return `${c.vehicleType === 'psv' ? c.chassisMake : c.make} - ${c.vehicleType === 'psv' ? c.chassisModel : c.model}`;
-  }
+    if (!(c as any)?.techRecord_make && !(c as any)?.techRecord_chassisMake) return '';
```

Comment:
Similar to comments elsewhere, can we think of a better name for this variable than 'C'.